sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/DigitalTwinsLifecycleSamples.cs

```diff
@@ -50,15 +49,15 @@ internal class DigitalTwinsLifecycleSamples
         private static readonly string s_twinsPath = Path.Combine(s_dtdlDirectoryPath, "DigitalTwins");
         private static readonly string s_relationshipsPath = Path.Combine(s_dtdlDirectoryPath, "Relationships");
 
-        private readonly string _eventhubEndpointName;
         private readonly string _eventRouteId = $"sampleEventRouteId-{Guid.NewGuid()}";
 
-        private DigitalTwinsClient DigitalTwinsClient { get; }
+        private readonly string eventhubEndpointName;
```

Comment:
why did we lose the `_` here?

Reply:
To make the variables used in sample code not look quite as foreign when seen in a snippet.